File: MPP_MusicFestival_Java/FestivalModel/src/main/java/domain/Entity.java
package domain;

import java.io.Serializable;

public class Entity<T> implements Serializable {

    public T Id;

    public Entity() {
    }

    public T getId() {
        return Id;
    }

    public void setId(T id) {
        Id = id;
    }

    @Override
    public String toString() {
        return Id.toString();

    }
}

File: model/Entity.cs


using System;

namespace MusicFestival.model
{
    [Serializable]
    public class Entity<T>
    {
        public T Id { get; set; }
        public Entity(T id)
        {
            this.Id = id;
        }

        public override string ToString()
        {
            return this.Id.ToString();
        }
    }
}

